### DISCUSSION TITLE
Running Stateless in Container

### DISCUSSION THREAD

**Discussion by SimmyD** [go-acme/lego#1769] [Q&A]
I am looking a storing certs (easy) and config(where I am struggling) in Vault/Consul so i have the ability to run a job in nomad every x amount of days to do a renewal on all my certs and so i don't need to worry on what host it runs on.

Traefik seems to do this in a way that it generates a acme-dns.json config file for that part and an acme.json that has the account info and all certs built into it as one file.  I am only looking at needing the account info stored there though as I would push the certs to a secret k/v in Vault for other servers to consume.

Is this possible and I cant just work it out?  or am I trying to do something stupid? :)

**Comment by ldez** [comment]
Hello,

I recommend posting in the [Traefik Community Forum](https://community.traefik.io/) because it's a question about Traefik.
